enterprise/tools/check_metrics/config.go

```diff
@@ -19,12 +22,17 @@ type PrometheusMetric struct {
 	// Set to override the default polling interval
 	PollingIntervalSeconds int `yaml:"polling_interval_seconds"`
 	// Set to override the default unhealthy count
-	MaxUnhealthyCount int `yaml:"max_unhealthy_count"`
+	MaxMetricPollUnhealthyCount int `yaml:"max_metric_poll_unhealthy_count"`
 	// If the metric does not meet this health threshold, it is considered unhealthy
 	HealthThreshold HealthThreshold `yaml:"health_threshold"`
 	// Set if it's valid for the metric value to be 0 or missing
 	// Ex. For error log count or invocation failure rate, at any given time the value can be 0
 	IsMissingDataValid bool `yaml:"is_missing_data_valid"`
+	// Primary metric failure should immediately result in a rollback
+	// If a secondary metric fails, it should only result in a rollback if several other secondary metrics
+	// are also failing (configurable by MaxSecondaryMetricFailureCount). This is intended to reduce unnecessary
+	// rollbacks from spiky secondary metrics
+	IsSecondaryMetric bool `yaml:"is_secondary_metric"`
```

Comment:
nit: feel free to shorten this to something like `is_secondary` or something if you want